# Install First

We need to install the libraries for todays presentation. 

In [1]:
import sys
!pip install matplotlib --user

You are using pip version 10.0.1, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.


"You are using pip version 18.0, however version 18.1 is available.

You should consider upgrading via the 'python -m pip install --upgrade pip' command."
- we are ignoring this for now.

In [3]:
import sys
!pip install numpy

You are using pip version 10.0.1, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.


In [ ]:
import sys
!pip install plotly --user

    99% |████████████████████████████████| 37.3MB 52.7MB/s eta 0:00:01��██▌            | 22.8MB 44.8MB/s eta 0:00:01   | 24.4MB 30.9MB/s eta 0:00:01█████████████▊         | 26.6MB 46.9MB/s eta 0:00:01███████▌       | 28.5MB 45.3MB/s eta 0:00:01████████████▉      | 30.2MB 40.3MB/s eta 0:00:01��█████████████████████████▌   | 33.3MB 27.7MB/s eta 0:00:01�▏ | 35.1MB 41.9MB/s eta 0:00:01

In [ ]:
import sys
!pip install altair --user

In [ ]:
import sys
!pip install pandas --user